pkg/blobfuse-proxy/server/server.go

```diff
@@ -62,16 +63,18 @@ func (server *MountServer) MountAzureBlob(ctx context.Context,
 
 	args := req.GetMountArgs()
 	authEnv := req.GetAuthEnv()
-	klog.V(2).Infof("received mount request: Mounting with args %v \n", args)
+	protocol := req.GetProtocol()
+	klog.V(2).Infof("received mount request: Protocol: %s, Mounting with args %v \n", protocol, args)
```

Comment:
it's better printing `server.blobfuseVersion` in the log for troubleshooting.

Reply:
Sure